src/Umbraco.Web.Common/ApplicationBuilder/UmbracoApplicationBuilder.cs

```diff
@@ -75,7 +75,7 @@ public void RegisterDefaultRequiredMiddleware()
 
         AppBuilder.UseUmbracoBackOfficeRewrites();
 
-        AppBuilder.UseStaticFiles();
+        AppBuilder.UseUmbracoStaticFiles();
```

Comment:
We can keep the existing `UseStaticFiles()` call that adds the default `StaticFileOptions` and configure it separately:
```c#
AppBuilder.Services.Configure<StaticFileOptions>(options => ...);
```

Or register an implementation of `IConfigureOptions<StaticFileOptions>` like the [example in our documentation](https://docs.umbraco.com/umbraco-cms/reference/response-caching#modify-the-cache-control-header-for-static-files). Also note that an existing `OnPrepareResponse` could potentially get overwritten, so we should capture the current callback and call that within our own to ensure both are executed:

```c#
Configure<StaticFileOptions>(options =>
{
    var onPrepareResponse = options.OnPrepareResponse;
    options.OnPrepareResponse = ctx =>
    {
        onPrepareResponse(ctx);

        // Additional logic
    };
});
```